docs/conf.py

```diff
@@ -15,155 +15,156 @@
 import itertools
 import datetime
 
-YEAR = datetime.date.today().strftime('%Y')
+YEAR = datetime.date.today().strftime("%Y")
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(".."))
 
 # -- General configuration -----------------------------------------------------
 
 # Use cython as the default syntax highlighting language, as python is a subset
 # this does the right thing
-highlight_language = 'cython'
+highlight_language = "cython"
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
+# needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
-    'sphinx.ext.imgmath',
-    'sphinx.ext.todo',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.autodoc',
-    'sphinx_issues',  # if this is missing, pip install sphinx-issues
-    'sphinx_tabs.tabs',  # if this is missing, pip install sphinx-tabs
-    ]
-
-try: import rst2pdf
-except ImportError: pass
-else: extensions.append('rst2pdf.pdfbuilder')
+    "sphinx.ext.imgmath",
+    "sphinx.ext.todo",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autodoc",
+    "sphinx_issues",  # if this is missing, pip install sphinx-issues
+    "sphinx_tabs.tabs",  # if this is missing, pip install sphinx-tabs
+    "sphinx_rtd_theme",
+]
+
+# try: import rst2pdf
+# except ImportError: pass
+# else: extensions.append('rst2pdf.pdfbuilder')
```

Comment:
Why did you remove this part?

Reply:
`readthedocs` have config for build `pdf` and `epub` in `.readthedocs.yaml`, and i have errors when try to build this locally. I will revert it back if I can't find ways to fix it, or many contributors need to build pdf locally